lib/temporal/workflow/context.rb

```diff
@@ -41,7 +41,7 @@ def execute_activity(activity_class, *input, **args)
 
         execution_options = ExecutionOptions.new(activity_class, options)
 
-        decision = Decision::ScheduleActivity.new(
+        workflow_task = WorkflowTask::ScheduleActivity.new(
```

Comment:
I think this is a wrong assumption — top level `Decision` is not the same as `DecisionTask`, so therefore it does not translate to `WorkflowTask`. This should be a `Command`, which is what a "decision" was used as a synonym of